content/chainguard/chainguard-images/getting-started/wordpress.md

```diff
@@ -0,0 +1,305 @@
+
+```yaml
+services:
+  app:
+    image: cgr.dev/chainguard/wordpress:latest-dev
+    restart: unless-stopped
+    environment:
+      WORDPRESS_DB_HOST: mariadb
+      WORDPRESS_DB_USER: $WORDPRESS_DB_USER
+      WORDPRESS_DB_PASSWORD: $WORDPRESS_DB_PASSWORD
+      WORDPRESS_DB_NAME: $WORDPRESS_DB_NAME
+    volumes:
+      - document-root:/var/www/html
+
+  nginx:
+    image: cgr.dev/chainguard/nginx
+    restart: unless-stopped
+    ports:
+      - 8000:8080
+    volumes:
+      - document-root:/var/www/html
+      - ./nginx.conf:/etc/nginx/nginx.conf
+
+  mariadb:
+    image: cgr.dev/chainguard/mariadb
+    restart: unless-stopped
+    environment:
+      MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: 1
+      MARIADB_USER: $WORDPRESS_DB_USER
+      MARIADB_PASSWORD: $WORDPRESS_DB_PASSWORD
+      MARIADB_DATABASE: $WORDPRESS_DB_NAME
+    ports:
+      - 3306:3306
+
+volumes:
+  document-root:
+```
+
+In this Docker Compose example, we define 3 services: `app`, `nginx`, and `mariadb`. Here's a breakdown of each service:
+
+- The `app` service uses the `latest-dev` variant of the Chainguard WordPress Image, and is configured to connect to the `mariadb` service. The entrypoint script in the WordPress image looks for environment variables to set up a custom `wp-config.php` file. The volume `document-root` defines a volume that will be shared between the `app` and the `nginx` services.
+- The `nginx` service uses the [Chainguard Nginx Image](https://images.chainguard.dev/directory/image/nginx/overview), and is configured to serve the WordPress application on port 8000.
+- The `mariadb` service uses the [Chainguard MariaDB Image](https://images.chainguard.dev/directory/image/mariadb/overview), and is configured with the necessary environment variables to create a database for the WordPress application.
+
+The environment variables used in this example are defined in a `.env` file located in the same directory as the `docker-compose.yaml` file:
+
+```ini
+WORDPRESS_DB_HOST=mariadb
+WORDPRESS_DB_USER=wp-user
+WORDPRESS_DB_PASSWORD=wp-password
+WORDPRESS_DB_NAME=wordpress
+```
+
+You can change these values to suit your needs. You can start the services by running:
+
+```bash
+docker-compose up
+```
+
+If you navigate to `http://localhost:8000` in your browser, you should access the WordPress installation page. Follow the on-screen instructions to complete the WordPress setup. Keep in mind that any customizations will be lost once the environment is turned down.
+
+To stop the services, type `CTRL+C` in the terminal where the services are running, and then run:
+
+```bash
+docker-compose down
+```
+
+This will remove the containers and networks created by the `docker-compose up` command. In the next example, we'll demonstrate how you can set up a volume with proper permissions to be able to persist customizations such as themes and plugins.
+
+## Example 2: Customizing a new WordPress Installation
```

Comment:
```suggestion
## Example 2: Customizing a New WordPress Installation
```
Minor, but I thought it looked better to have "New" capitalized here, since it's an adjective